lms/static/sass/course/modules/_student-notes.scss

```diff
@@ -68,8 +101,190 @@
+      background-position: 0 0;
+    }
+  }
+}
+
+// editing
+.annotator-editor {
+
+  .annotator-controls {
+    @include text-align(left);
+    @include clearfix();
+    background: $notes-annotator-background-med !important; //matches annotator JS editing bubble triangle color
+    font-family: $f-sans-serif !important;
+    padding: 8px;
+    border: none !important;
+    border-radius: 0 !important;
+
+    // actions
+    .annotator-save, .annotator-cancel {
+      @extend %notes-reset-background;
+      font-family: $f-sans-serif !important;
+      font-size: 14px !important;
+      padding: ($baseline/4) ($baseline/2) !important;
+      border: none;
+      box-shadow: none;
+      text-shadow: none !important;
+
+      // removing vendor icons
+      &:after {
+        display: none !important;
+      }
+    }
+
+    .annotator-save {
+      @include float(left);
+    }
+
+    .annotator-cancel {
+      background-color: $transparent !important;
+    }
+  }
+
+  .annotator-item {
+
+    textarea {
+      @extend %notes-reset-font;
+      @extend %t-demi-strong;
+      padding: ($baseline/5) !important;
+      font-size: 14px !important;
+      line-height: 22px !important;
+      color: $gray-d3 !important;
+      background: $notes-annotator-background-light !important; //matches annotator JS editing bubble triangle color
+
+      // STATE: hover/focus
+      &:hover, &:focus {
+        background: $notes-annotator-background-light;
+      }
+    }
+  }
+}
+
+
+// +listing notes (overrides for vendor styling)
+// --------------------
+// highlight
+.annotator-hl {
+  background: $student-notes-highlight-color-focus;
+}
+
+// content
+.annotator-viewer {
+
+  // poorly scoped selector for content of a note's comment
+  div:first-of-type {
+    @extend %notes-reset-font;
+    padding: ($baseline/4) !important;
+    font-size: 14px !important;
```

Comment:
NOTE: wish we could use our ``font-size()`` mixin, but alas, we need the ``!important``.